### ISSUE TITLE
planet failed to start, switched to edge, docker network error logs

### ISSUE THREAD

**Issue by SaulMoonves** [Native-Planet/GroundSeg#627]
This may be related to #619 but here are my logs after restarting GroundSeg:

```
Dec 08 00:19:39 nativeplanet groundseg[3990]:                                        !G#:
Dec 08 00:19:39 nativeplanet groundseg[3990]:                                     .7G@@@^
Dec 08 00:19:39 nativeplanet groundseg[3990]:           .                       :J#@@@@P.
Dec 08 00:19:39 nativeplanet groundseg[3990]:      .75GB#BG57.                ~5&@@@&Y^
Dec 08 00:19:39 nativeplanet groundseg[3990]:     ?&@@@@@@@@@&J             !G@@@@B?. .^
Dec 08 00:19:39 nativeplanet groundseg[3990]:    Y@@@@@@@@@@@@@J         :?B@@@@G!  :Y&&:
Dec 08 00:19:39 nativeplanet groundseg[3990]:    @@@@@@@@@@@@@@B       ^Y&@@@&5^  ~P&@@@:
Dec 08 00:19:39 nativeplanet groundseg[3990]:    Y@@@@@@@@@@@@@J     !P&@@@#J.  7B@@@@G!
Dec 08 00:19:39 nativeplanet groundseg[3990]:     ?#@@@@@@@@@&?   .7B@@@@G7  :J#@@@&5~
Dec 08 00:19:39 nativeplanet groundseg[3990]:      .!YGBBBGY7.  :J#@@@&P~  ^5&@@@#J:  !?.
Dec 08 00:19:39 nativeplanet groundseg[3990]:                 ^5&@@@#J:  !G@@@@G7. .?B@@:
Dec 08 00:19:39 nativeplanet groundseg[3990]:              .7G@@@@G7. :J#@@@&P~  ^Y#@@@&:
Dec 08 00:19:39 nativeplanet groundseg[3990]:             .P&&&&G!   ~B&&&#5^   ~#&&&&&P.
Dec 08 00:19:39 nativeplanet groundseg[3990]: Ｎａｔｉｖｅ Ｐｌａｎｅｔ
Dec 08 00:19:39 nativeplanet groundseg[3990]:  ▄▄ • ▄▄▄        ▄• ▄▌ ▐ ▄ ·▄▄▄▄  .▄▄ · ▄▄▄ . ▄▄ • 𝐯𝟐!
Dec 08 00:19:39 nativeplanet groundseg[3990]: ▐█ ▀ ▪▀▄ █·▪     █▪██▌•█▌▐███▪ ██ ▐█ ▀. ▀▄.▀·▐█ ▀ ▪
Dec 08 00:19:39 nativeplanet groundseg[3990]: ▄█ ▀█▄▐▀▀▄  ▄█▀▄ █▌▐█▌▐█▐▐▌▐█· ▐█▌▄▀▀▀█▄▐▀▀▪▄▄█ ▀█▄ 🪐
Dec 08 00:19:39 nativeplanet groundseg[3990]: ▐█▄▪▐█▐█•█▌▐█▌.▐▌▐█▄█▌██▐█▌██. ██ ▐█▄▪▐█▐█▄▄▌▐█▄▪▐█
Dec 08 00:19:39 nativeplanet groundseg[3990]: ·▀▀▀▀ .▀  ▀ ▀█▄▀▪ ▀▀▀ ▀▀ █▪▀▀▀▀▀•  ▀▀▀▀  ▀▀▀ ·▀▀▀▀ (~)
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.514294673-08:00","level":"INFO","msg":"Starting GroundSeg"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.51674763-08:00","level":"INFO","msg":"Urbit is love <3"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.516812788-08:00","level":"INFO","msg":"Thank you for supporting Native Planet!"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.520628259-08:00","level":"INFO","msg":"Fixer cron found. Doing nothing"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.520713642-08:00","level":"INFO","msg":"Loading configs from /opt/nativeplanet/groundseg"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.58784055-08:00","level":"INFO","msg":"Binary sha256 hash: 6fbf7790fd706ecb0574a8d964fd0cc6710c12987b154a2b620e641618887dd9"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.593717173-08:00","level":"INFO","msg":"Successfully stopped container groundseg-webui\n"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.603506666-08:00","level":"INFO","msg":"Docker version: 24.0.7"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.60354605-08:00","level":"INFO","msg":"Bootstrapping state"}
Dec 08 00:19:40 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:40.603554773-08:00","level":"INFO","msg":"Resolving pier status"}
Dec 08 00:19:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:41.610853819-08:00","level":"INFO","msg":"Retrieving StarTram region info"}
Dec 08 00:19:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:41.854154969-08:00","level":"INFO","msg":"Internet available: true"}
Dec 08 00:19:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:41.854257172-08:00","level":"INFO","msg":"Setting up swap /opt/nativeplanet/groundseg/swapfile for 16G"}
Dec 08 00:19:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:41.957561259-08:00","level":"INFO","msg":"StarTram info retrieved"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.032218157-08:00","level":"INFO","msg":"Version info retrieved"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.032415895-08:00","level":"INFO","msg":"Downloading Wireguard image"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041056479-08:00","level":"INFO","msg":"Loading MinIO containers"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041457761-08:00","level":"INFO","msg":"Llama GPT disabled!"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041185088-08:00","level":"INFO","msg":"Loading Urbit ships"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041575016-08:00","level":"INFO","msg":"Loading pier molten-magnum"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041880202-08:00","level":"INFO","msg":"Loading MC container"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041982263-08:00","level":"INFO","msg":"Running MC"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041102588-08:00","level":"INFO","msg":"Loading NetData container"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.041843499-08:00","level":"INFO","msg":"Loading Startram Wireguard container"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.042905194-08:00","level":"INFO","msg":"Updating WG config"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.043162552-08:00","level":"INFO","msg":"Running Wireguard"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.043340693-08:00","level":"INFO","msg":"Running NetData"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.193228165-08:00","level":"ERROR","msg":"Error starting molten-magnum: Error response from daemon: No such container: aaed63469a126093e7606ec3c78b4574d19e5b331a7e63d197085dab80ee7321"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.515598395-08:00","level":"INFO","msg":"Started stopped container wireguard"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.515635364-08:00","level":"INFO","msg":"Docker: wireguard started"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.516255289-08:00","level":"INFO","msg":"wireguard state:<hidden>"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.857134321-08:00","level":"INFO","msg":"minio_molten-magnum started with image registry.hub.docker.com/minio/minio:latest@sha256:6d6cf693fd70ca6e15709fa44d39b44f98fc5b58795697486a95ac1cc2ad9880"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.857160273-08:00","level":"INFO","msg":"Docker: minio_molten-magnum started"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.875753745-08:00","level":"INFO","msg":"Started stopped container mc"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.87576045-08:00","level":"INFO","msg":"Docker: mc started"}
Dec 08 00:19:42 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:42.876434399-08:00","level":"INFO","msg":"mc state:<hidden>"}
Dec 08 00:19:43 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:43.40711585-08:00","level":"INFO","msg":"fFG0Bkm-zOiCToMpAKteZhCu6oJsUX3o24xxT31Tq1w= added to auth"}
Dec 08 00:19:44 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:44.069644698-08:00","level":"INFO","msg":"minio_molten-magnum state:<hidden>"}
Dec 08 00:19:49 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:49.946273398-08:00","level":"WARN","msg":"Docker: netdata died!"}
Dec 08 00:19:50 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:50.193994472-08:00","level":"ERROR","msg":"Error starting NetData: Error response from daemon: driver failed programming external connectivity on endpoint netdata (79680ff124e084c4e8ed8400e6afba0a61e44f4b2dba51540617c9643864a91f):  (iptables failed: iptables --wait -t nat -A DOCKER -p tcp -d 0/0 --dport 19999 -j DNAT --to-destination 172.17.0.2:19999 ! -i docker0: iptables: No chain/target/match by that name.\n (exit status 1))"}
Dec 08 00:19:50 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:19:50.194088629-08:00","level":"ERROR","msg":"Unable to load Netdata! Error response from daemon: driver failed programming external connectivity on endpoint netdata (79680ff124e084c4e8ed8400e6afba0a61e44f4b2dba51540617c9643864a91f):  (iptables failed: iptables --wait -t nat -A DOCKER -p tcp -d 0/0 --dport 19999 -j DNAT --to-destination 172.17.0.2:19999 ! -i docker0: iptables: No chain/target/match by that name.\n (exit status 1))"}
Dec 08 00:20:21 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:20:21.220164896-08:00","level":"INFO","msg":"fFG0Bkm-zOiCToMpAKteZhCu6oJsUX3o24xxT31Tq1w= added to auth"}
Dec 08 00:20:26 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:20:26.462612439-08:00","level":"INFO","msg":"Urbit"}
Dec 08 00:20:26 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:20:26.469212095-08:00","level":"ERROR","msg":"Click |exit for molten-magnum failed to get exec: Container not found"}
Dec 08 00:20:26 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:20:26.475421471-08:00","level":"INFO","msg":"Successfully stopped container molten-magnum\n"}
Dec 08 00:23:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:23:41.862229887-08:00","level":"WARN","msg":"Couldn't get network for molten-magnum"}
Dec 08 00:24:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:24:41.873722446-08:00","level":"WARN","msg":"Couldn't get network for molten-magnum"}
Dec 08 00:25:41 nativeplanet groundseg[3990]: {"time":"2023-12-08T00:25:41.883055495-08:00","level":"WARN","msg":"Couldn't get network for molten-magnum"}
```

here's a list of my containers:

```
784e46b74390   registry.hub.docker.com/netdata/netdata:latest         "/usr/sbin/run.sh"       18 minutes ago   Created                               netdata
a9e396771ec8   registry.hub.docker.com/minio/mc:latest                "/bin/bash"              19 minutes ago   Up 19 minutes                         mc
d772f5d450c0   registry.hub.docker.com/minio/minio:latest             "minio server /data …"   19 minutes ago   Up 19 minutes                         minio_molten-magnum
bd124d62f6d2   registry.hub.docker.com/linuxserver/wireguard:latest   "/init"                  19 minutes ago   Up 19 minutes             51820/udp   wireguard
1ce54896456c   registry.hub.docker.com/nativeplanet/urbit:v2.12       "bash /urbit/start_u…"   19 minutes ago   Created                               molten-magnum
5e6bac72b01e   nativeplanet/groundseg-webui:latest                    "node index.js"          3 days ago       Exited (137) 3 days ago               groundseg-webui
```

trying to turn "power" on in GroundSeg does nothing.

Device: Tellurian 1.0

Not quite fresh install. I started up everything for the first time a few days ago and did the install process and it worked for a little while and then had multiple problems. I've tried rebuilding one time before switching to edge (trying to help you guys.) I started from a freshly breached planet using a key file.

halp

**Comment by SaulMoonves** [comment]
I'd like a working planet sooner rather than later but if I can help you diagnose this in any way I'll prioritize helping you, I can live without my planet for a while to help make GroundSeg more robust.

**Comment by SaulMoonves** [comment]
remote checkbox is on. wifi stopped working yesterday so I plugged in a cable and rebooted. I am using Anchor rather than StarTram.

Oh, I just tried execing into the wireguard container and it doesn't have the wireguard interface. I tried starting it and got this:

```
root@wireguard:/etc/wireguard# wg-quick up wg0
Warning: `/config/wg0.conf' is world accessible
[#] ip link add wg0 type wireguard
[#] wg setconf wg0 /dev/fd/63
Temporary failure in name resolution: `anchor.molten-magnum.com:51820'. Trying again in 1.00 seconds...
Temporary failure in name resolution: `anchor.molten-magnum.com:51820'. Trying again in 1.20 seconds...
```

here are the nameservers:

nameserver 8.8.8.8
nameserver 8.8.4.4

pinging the nameservers gets 100% packet loss.

```
root@wireguard:/etc/wireguard# ip addr show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
7: eth0@if8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    link/ether 02:42:ac:11:00:03 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 172.17.0.3/16 brd 172.17.255.255 scope global eth0
       valid_lft forever preferred_lft forever
```

**Comment by yapishu** [comment]
@SaulMoonves Can you try running `sudo systemctl enable systemd-resolved`, then rebooting your box? We had a nasty bug last week that could happen after network loss that this sounds like. Additionally, try hitting the 'rebuild' button at the bottom of your ship's container once it's back up

**Comment by SaulMoonves** [comment]
wireguard is working now, thanks. I tried rebuilding my planet and it failed:

```
Dec 09 01:42:51 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:42:51.510207986-08:00","level":"INFO","msg":"Starting logs for molten-magnum"}
Dec 09 01:42:57 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:42:57.285930707-08:00","level":"INFO","msg":"Urbit"}
Dec 09 01:42:57 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:42:57.292177821-08:00","level":"ERROR","msg":"Click |exit for molten-magnum failed to get exec: Container not found"}
Dec 09 01:42:57 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:42:57.298854045-08:00","level":"INFO","msg":"Successfully stopped container molten-magnum\n"}
Dec 09 01:43:38 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:43:38.222883101-08:00","level":"INFO","msg":"Urbit"}
Dec 09 01:43:38 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:43:38.244154409-08:00","level":"INFO","msg":"Deleted Container: molten-magnum"}
Dec 09 01:43:41 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:43:41.246004474-08:00","level":"ERROR","msg":"Failed to rebuild container molten-magnum: molten-magnum marked noboot!"}
Dec 09 01:43:52 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:43:52.558972953-08:00","level":"INFO","msg":"<REDACTED> added to auth"}
Dec 09 01:44:32 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:44:32.406922809-08:00","level":"INFO","msg":"Starting logs for molten-magnum"}
Dec 09 01:44:32 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:44:32.408253835-08:00","level":"ERROR","msg":"Error streaming previous logs: Error response from daemon: No such container: molten-magnum"}
Dec 09 01:44:40 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:44:40.190994871-08:00","level":"INFO","msg":"Urbit"}
Dec 09 01:44:40 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:44:40.19285344-08:00","level":"ERROR","msg":"Failed to delete container molten-magnum"}
Dec 09 01:44:43 nativeplanet groundseg[1331]: {"time":"2023-12-09T01:44:43.194449181-08:00","level":"ERROR","msg":"Failed to rebuild container molten-magnum: molten-magnum marked noboot!"}
```

here's my docker containers:

```
root@nativeplanet:~# docker ps
CONTAINER ID   IMAGE                                                  COMMAND                  CREATED         STATUS         PORTS       NAMES
8dd6235d9315   registry.hub.docker.com/minio/mc:latest                "/bin/bash"              4 minutes ago   Up 4 minutes               mc
9644774557cf   registry.hub.docker.com/minio/minio:latest             "minio server /data …"   4 minutes ago   Up 4 minutes               minio_molten-magnum
176fd8573118   registry.hub.docker.com/linuxserver/wireguard:latest   "/init"                  5 minutes ago   Up 4 minutes   51820/udp   wireguard
```

**Comment by yapishu** [comment]
@SaulMoonves I figured out the issue with ships failing to rebuild, Docker network issues due to overly aggressive C2C mode, both now fixed as of [ v2.0.11](https://github.com/Native-Planet/GroundSeg/releases/tag/v2.0.11)
